Description: Upstream changes introduced in version 1.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 kvp-server (1.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
 .
 The person named in the Author field signed this changelog entry.
Author: Dmitry <maintainer@email.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- kvp-server-1.0.orig/tests/ut_logregthread/ut_logregthread.pro
+++ kvp-server-1.0/tests/ut_logregthread/ut_logregthread.pro
@@ -1,5 +1,7 @@
 include(../common_tests.pri)
 TARGET = ut_logregthread
+target.path = /usr/share/bin
+INSTALLS += target
 SOURCES += \
 ut_logregthread.cpp \
 $$SRCDIR/logregthread.cpp\
--- kvp-server-1.0.orig/tests/ut_message/ut_message.pro
+++ kvp-server-1.0/tests/ut_message/ut_message.pro
@@ -1,5 +1,7 @@
 include(../common_tests.pri)
 TARGET = ut_message
+target.path = /usr/share/bin
+INSTALLS += target
 SOURCES += \
 ut_message.cpp \
 $$SRCDIR/message.cpp
--- kvp-server-1.0.orig/tests/ut_contactsbase/ut_contactsbase.pro
+++ kvp-server-1.0/tests/ut_contactsbase/ut_contactsbase.pro
@@ -3,6 +3,8 @@
 # -------------------------------------------------
 include(../common_tests.pri)
 TARGET = ut_contactsbase
+target.path = /usr/share/bin
+INSTALLS += target
 SOURCES += \
 ut_contactsbase.cpp\
 $$SRCDIR/contactsbase.cpp
--- kvp-server-1.0.orig/tests/ut_activeclients/ut_activeclients.pro
+++ kvp-server-1.0/tests/ut_activeclients/ut_activeclients.pro
@@ -1,5 +1,7 @@
 include(../common_tests.pri)
 TARGET = ut_activeclients
+target.path = /usr/share/bin
+INSTALLS += target
 SOURCES += \
 ut_activeclients.cpp \
 $$SRCDIR/activeclients.cpp
